avocado/core/extension_manager.py

```diff
@@ -61,7 +61,7 @@ def __init__(self, namespace, invoke_kwds=None):
             try:
                 plugin = ep.load()
                 obj = plugin(**invoke_kwds)
-            except Exception as exception:
```

Comment:
We need a `# pylint: disable=W0703` here instead, as the plugin instantiation can fail due to many other causes, not only an `ImportError`.